ministryofjustice/laa-review-criminal-legal-aid
# Update copy and layout for codefendants

## Description of change
Updates codefendant table format and copy. Explicitly says if there is a conflict or not in the caption text, and adds badge in No conflict situations
## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAP-561
## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:
<img width="794" alt="Screenshot 2023-08-21 at 15 12 54" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/45827968/87bb9896-af0d-463b-8e74-d30f985bdc00">

### After changes:
<img width="836" alt="image" src="https://github.com/ministryofjustice/laa-review-criminal-legal-aid/assets/45827968/1af357af-4ca1-418a-89ce-ca3ec6251091">

## How to manually test the feature
Create a case with two codefendants, one with a conflict and one with no conflict.
Check the table looks right. Create a case with no codefendants, make sure table remains how it always has been
